charts/tfy-karpenter-config/values.yaml

```diff
@@ -227,6 +227,17 @@ karpenter:
           echo "CONTAINERD_VERSION is empty or not within the specified range."
       fi
 
+      rmmod nvidia_drm
+      rmmod nvidia_modeset
+      rmmod nvidia_uvm
+      rmmod nvidia
+      echo "Writing NVreg_EnableGpuFirmware=0 to /etc/modprobe.d/nvidia.conf"
+      echo "options nvidia NVreg_EnableGpuFirmware=0" | tee --append /etc/modprobe.d/nvidia.conf
+      echo "Writing NVreg_EnableGpuFirmware=0 to /etc/modprobe.d/nvidia-gsp.conf"
+      echo "options nvidia NVreg_EnableGpuFirmware=0" | tee --append /etc/modprobe.d/nvidia-gsp.conf
+      echo "Running dracut"
+      dracut -f
```

Comment:
This will have no effect until reboot if the module was not unloaded beforehand

Reply:
Is EKS fine with this?
How much latency are we adding here?

Reply:
> Is EKS fine with this?
Does not cause any problems as such but the problem is this works now but if some config changes under our feet then it would stop working. 


> How much latency are we adding here?
Maybe 1-2s, I can measure it out